src/agent/Cargo.toml

```diff
@@ -55,7 +55,9 @@ procfs = "0.12.0"
 
 anyhow = "1"
 
-cgroups = { package = "cgroups-rs", git = "https://github.com/kata-containers/cgroups-rs", rev = "v0.3.5" }
+# TODO
```

Comment:
Is this ready for review or you still testing?

Reply:
Nope, I am working on fixing CI issues. I've changed the status to drafted.